runner/test/cases/server/plugins/engine/pageControllers/getConditionEvaluationContext.test.ts

```diff
@@ -0,0 +1,79 @@
+import * as Code from "@hapi/code";
+import * as Lab from "@hapi/lab";
+import formJson from "../../../../../../src/server/forms/get-condition-evaluation-context.json";
+
+import { FormModel } from "../../../../../../src/server/plugins/engine/models";
+import { PageController } from "server/plugins/engine/pageControllers";
+
+const lab = Lab.script();
+exports.lab = lab;
+const { expect } = Code;
+const { describe, suite, test } = lab;
+
+suite("Condition Evaluation Context", () => {
+  describe("getConditionEvaluationContext", () => {
+    test("it correctly includes/filters state values", () => {
```

Comment:
no need to action, but no need to nest these if you've only really got the one case. you can also use `it` 

```suggestion
const { it, describe } = lab;

describe("getConditionEvaluationContext", () => {
  it("correctly includes/filters state values", () => {
```

Reply:
Funny, I did try "it" previously but it wouldn't run my test so I gave up. Your code example seems to work though so thanks!